dockerfiles/centos-stream8-vdbench5.04.07-container-disk/state_signals_responder.py

```diff
@@ -0,0 +1,63 @@
+
+import sys
+import time
+import state_signals
+from multiprocessing import Process
+import argparse
+import subprocess
+
+
+class StateSignalsResponder:
+    """
+    This class run state signals responder
+    """
+    def __init__(self, redis_host: str, run_workload_method: str, timeout: int):
+        self.__redis_host = redis_host
+        self.__run_workload_method = run_workload_method
+        self.__timeout = timeout
+
+    def _listener(self):
+        time.sleep(5)
```

Comment:
Why?

Reply:
As I said before, its need to wait between fetch

Reply:
Why is that?  If it's implemented correctly, there should be no need to wait, and this kind of sleep is prone to race conditions.

Reply:
Remove it